spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/ScopePassingSpanSubscriber.java

```diff
@@ -115,6 +115,9 @@ public Object scanUnsafe(Attr key) {
 		if (key == Attr.PARENT) {
 			return this.s;
 		}
+		else if (key == Attr.RUN_STYLE) {
```

Comment:
I wonder if there's an errorprone or other way to scan a project for a missing "if" like this (similar to missing branch in a case)...